x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/utils.ts

```diff
@@ -105,3 +117,77 @@ export const showErrorToast = (
     toasts.addDanger(extractErrorMessage(error, FETCH_GRAPH_FAILED_TEXT));
   }
 };
+
+export const buildGraphFromViewModels = (
```

Comment:
nit, if it is only used tests preparation it is worth moving it to one of the test file code